### ISSUE TITLE
Lifter Request for Oleksii Melnyk

### ISSUE THREAD

**Issue by TrystanOakley** [sstangl/openpowerlifting#502]
http://www.openpowerlifting.org/lifters.html?q=Oleksii%20Melnyk

He says we are missing a meet that was reported here http://en.allpowerlifting.com/lifters/UKR/melnyk-oleksiy-19960/

I know we are not supposed to link to all powerlifting but I could not find the original meet

**Comment by TrystanOakley** [comment]
The federation is here http://www.wpuf.org but I can not find the meet or any meet results for that matter

**Comment by sstangl** [comment]
Yeah, we basically do not support Russian and Ukrainian federations at the moment.

That is somewhat intentional, since:
1. They already have a pretty good site for meet results -- allpowerlifting.com;
2. Their meet results are posted in very annoying formats;
3. Their lifters basically only check allpowerlifting, and AFAIK don't care much about non-Russian/Ukranian lifters

So it's low priority. In my mind, Russia support comes after Australia, where the situation is much more favorable to us. But if we can hunt down these meets, of course we'll add them.

**Comment by TrystanOakley** [comment]
Got it.

**Comment by sstangl** [comment]
Don't, uh, tell people that, obviously! If we can add their meets, we should. It will just take us quite a while to get to actively tracking them.

**Comment by TrystanOakley** [comment]
Haha yeah no worries I gotcha
